RELEASE.txt

```diff
@@ -71,22 +67,14 @@ access), specifically to:
   `NumpyVersion <https://github.com/scikit-image/scikit-image/pull/4947>`_
   correctly interprets the version number.
 
- - On the main branch, edit ``doc/source/_static/docversions.js``,
+- On the main branch, edit ``doc/source/_static/docversions.js``,
   add the release, e.g., `0.15.x`, and commit.  Cherry-pick this
   change back to the release branch.
 
-- Make sure you have all the required write permissions (if not, you will
-  need to ask Stéfan or Juan to grant you the access), specifically, to:
-
-    - https://github.com/scikit-image/scikit-image-wheels
-    - https://pypi.org/project/scikit-image/
-    - https://github.com/scikit-image/scikit-image-web
-
-
 - Perform the following step first on a release candidate to check the wheels
   are building fine. Then proceed with the release.
 
-    1. Back on the release branch, update the version number to stable in
+    1. Update the version number to stable in
```

Comment:
is this automated now? can it be automated?

Reply:
Yes it can be automated, but I would prefer to have that as part of the release process as cibuildwheels uses that version string to generate the wheels files

Reply:
Ok a pr once towncrier is done would be good.